services/api-gateway/tools/templates/test_endpoints.py.jinja2

```diff
@@ -0,0 +1,61 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from typing import List
+
+import pytest
+from starlette.testclient import TestClient
+from starlette import status
+
+
+# TODO: app is init globally ... which is bad!
+from simcore_service_api_gateway.main import api_version, app, api_version_prefix
+
+
+@pytest.fixture
+def client(environ_context, postgres_service):
+    # TODO: create new web-app everyt
+    with TestClient(app) as cli:
+        yield cli
+
+
+def test_list_{{ rnp }}(client):
+    response = client.get(f"/{api_version_prefix}/{{ rnp }}")
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == []
+
+    # inject three dagin
+    response = client.get(f"/{api_version_prefix}/{{ rnp }}")
+    assert response.status_code == status.HTTP_200_OK
+    # TODO: assert i can list them as dagouts
+
```

Comment:
need some explanations on the templates...

Reply:
This is still work in progress